data/cncf.yaml

```diff
@@ -45,6 +51,12 @@
       check_sets:
         - community
         - code
+        - docs
```

Comment:
The `docs` check set is a subset of the `code` check set, so it isn't needed in this case. Please remove it.